docs/ispc.rst

```diff
@@ -715,6 +715,11 @@ Updating ISPC Programs For Changes In ISPC 1.27.0
 Added cross lane operations for unsigned types: ``broadcast``, ``rotate``, ``shift``,
 and ``shuffle``.
 
+``max`` function for short vectors of basic types was added to the standard
```

Comment:
I'd write here that it was added experimentally and will be extended with other functions which are used often with short vectors later.
Do you plan to add other functions for the next release?

Reply:
I think that adding more or less all element-wise functions needs to be done for the next release.